releases.go

```diff
@@ -562,11 +562,85 @@ func listReleases(c *gin.Context) {
 	namespace := c.Param("namespace")
 	kubeContext := c.Query("kube_context")
 	var options releaseListOptions
-	err := c.ShouldBindJSON(&options)
-	if err != nil && err != io.EOF {
+	var err error
+	options.All, err = strconv.ParseBool(c.DefaultQuery("all", "false"))
+	if err != nil {
+		respErr(c, err)
+		return
+	}
+
+	options.AllNamespaces, err = strconv.ParseBool(c.DefaultQuery("all_namespaces", "false"))
+	if err != nil {
+		respErr(c, err)
+		return
+	}
+
+	options.ByDate, err = strconv.ParseBool(c.DefaultQuery("by_date", "false"))
+	if err != nil {
+		respErr(c, err)
+		return
+	}
+
+	options.SortReverse, err = strconv.ParseBool(c.DefaultQuery("sort_reverse", "false"))
+	if err != nil {
+		respErr(c, err)
+		return
+	}
+
+	options.Limit, err = strconv.Atoi(c.DefaultQuery("limit", "0"))
+	if err != nil {
+		respErr(c, err)
+		return
+	}
+
+	options.Offset, err = strconv.Atoi(c.DefaultQuery("offset", "0"))
+	if err != nil {
+		respErr(c, err)
+		return
+	}
+
+	options.Filter = c.Query("filter")
```

Comment:
According to the documentation, default is an empty string. Does that matter?

Reply:
If the filter param is not set `c.Query` will return ""